.changelog/epilogue.md

```diff
@@ -0,0 +1,618 @@
+# CHANGELOG
```

Comment:
What's the difference/purpose of `.changelog/epilogue.md` compared to `CHANGELOG.md`?

Reply:
`unclog` treats the `epilogue` as `Any content to be added to the end of the generated CHANGELOG`. So unclog just copies the contents of `.changelog/epilogue.md` to the end of the changelog. This is helpful when generating a new changelog because we don't have to maintain `.changelog` entries for older versions.